### ISSUE TITLE
can't launch on archlinux (kde)

### ISSUE THREAD

**Issue by bachy** [mapmapteam/mapmap#458]
## Expected Behavior
launch the app with cli
```
./mapmap
```

## Actual Behavior
nothing, mapmap is runing, htop is seeing the processes, but no UI appears, no error message ...
i can do ``` ./mapmap -h``` or ```./mapmap -v```
```shell
$ ./mapmap -v
MapMap 0.6.2
```

## Steps to Reproduce the Problem

  1. install dependencies and build the app
  2. launch executable mapmap by cli

## Specifications

  - Version: 6.02
 - Platform: archlinux (fully updated)

**Comment by baydam** [comment]
Hello! Did you try to pull last change front github + clean?
You can also try ./mapmap -R

**Comment by bachy** [comment]
yes i started to build the last commit from master, then seeing that it was not working as decribed before, i checkedout 0.6.2 tag.
and yes i tried in both case to run ```./mapmap -R```
i also tried from 0.6.2 tar.gz release

**Comment by baydam** [comment]
Please try to compile the version from `develop` branch and let us know
